### ISSUE TITLE
Install Accessible Divi plug-in.

### ISSUE THREAD

**Issue by getorganizedbk** [GetOrganizedBrooklyn/getorganizedbk-website#3]
Install Accessible Divi plug-in to troubleshoot a range of issues with the current website: https://wordpress.org/plugins/accessible-divi/

* Adds appropriate ARIA attributes
* Improved keyboard navigation in menus
* Fixes missing & incorrectly assigned labels
* Makes modules such as Toggle & Accordion focusable and keyboard interactive
* Adds a visual outline to focusable elements for keyboard only navigation
* Ability to change keyboard only outline color
* Fixes Divi incorrectly using screen reader classes
* Adds skip navigation link optimized for Divi markup
* Hide icons from screen readers which can affect reading of text
* Fix duplicate menu IDs
* Tota11y integration

This is especially critical for keyboard users, who currently cannot see what is in focus throughout the site.
